udata/core/organization/factories.py

```diff
@@ -13,8 +13,8 @@ class OrganizationFactory(MongoEngineFactory):
     class Meta:
         model = models.Organization
 
-    name = factory.LazyAttribute(lambda o: faker.sentence())
-    description = factory.LazyAttribute(lambda o: faker.text())
+    name = factory.LazyAttribute(lambda o: faker.text())
+    description = factory.LazyAttribute(lambda o: faker.sentence())
```

Comment:
Reverse comment of the above one: `faker.sentence()` output a single sentence whereas `faker.text()` give us an entire paragraph which is what we expect for a description.